jsk_rviz_plugins/CMakeLists.txt

```diff
@@ -170,14 +170,7 @@ if(rviz_QT_VERSION VERSION_LESS "5")
 else()
   target_link_libraries(jsk_rviz_plugins Qt5::Widgets ${catkin_LIBRARIES} ${rviz_DEFAULT_PLUGIN_LIBRARIES})
 endif()
-add_dependencies(jsk_rviz_plugins
-  jsk_footstep_msgs_gencpp
-  jsk_gui_msgs_gencpp
-  jsk_hark_msgs_gencpp
-  jsk_recognition_msgs_gencpp
-  people_msgs_gencpp
-  view_controller_msgs_gencpp
-  ${PROJECT_NAME}_gencpp)
+add_dependencies(jsk_rviz_plugins ${PROJECT_NAME}_generate_messages_cpp)
```

Comment:
if some of jsk_rviz_plugins sources depends on these messages, we need add them within `generate_messages` or `add_dependencies people_msgs_generate_messages_cpp...`
https://github.com/jsk-ros-pkg/jsk_visualization/pull/692/files#diff-9f265c46d6e262161a6ffa5b777d3534R39